docs/api/in-app-purchase.md

```diff
@@ -21,10 +21,11 @@ Returns:
 
 The `inAppPurchase` module has the following methods:
 
-### `inAppPurchase.purchaseProduct(productID[, quantity])`
+### `inAppPurchase.purchaseProduct(productID[, quantity, userID])`
 
 * `productID` string - The identifiers of the product to purchase. (The identifier of `com.example.app.product1` is `product1`).
 * `quantity` Integer (optional) - The number of items the user wants to purchase.
+* `userID` string (optional) - The string that associates the transaction with a user account on your service (applicationUsername).
```

Comment:
This should probably be called `username` to match the underlying concept from the StoreKit API.